product-scenarios/1-manage-users-and-roles/1.1-admin-user-manage-roles/1.1.2-manage-roles-with-scim1/src/test/java/org/wso2/identity/scenarios/test/scim/ManageSecondaryUserStoreSCIM1TestCase.java

```diff
@@ -0,0 +1,169 @@
+
+        this.username = username;
+        this.password = password;
+        this.tenantDomain = tenantDomain;
+        this.inputFileName = inputFile;
+    }
+
+    @BeforeClass(alwaysRun = true)
+    public void testInit() throws Exception {
+
+        super.init();
+        scim1Group = new SCIM1CommonClient(getDeploymentProperty(IS_HTTPS_URL));
+        loginAndObtainSessionCookie();
+        userStoreConfigAdminServiceClient = new UserStoreConfigAdminServiceClient(backendServiceURL, sessionCookie);
+        client = HttpClients.createDefault();
+    }
+
+    @Test
+    private void addSecondaryUserStore() throws Exception {
+
+        String jdbcClass = "org.wso2.carbon.user.core.jdbc.JDBCUserStoreManager";
+        H2DataBaseManager dbmanager = new H2DataBaseManager("jdbc:h2:./repository/database/" +
+                USER_STORE_DB_NAME, DB_USER_NAME, DB_USER_PASSWORD);
+        dbmanager.disconnect();
+
+        PropertyDTO[] propertyDTOs = new PropertyDTO[10];
+        for (int i = 0; i < 10; i++) {
+            propertyDTOs[i] = new PropertyDTO();
+        }
+
+        propertyDTOs[0].setName("driverName");
+        propertyDTOs[0].setValue("org.h2.Driver");
+
+        propertyDTOs[1].setName("url");
+        propertyDTOs[1].setValue("jdbc:h2:repository/database/" + USER_STORE_DB_NAME);
+
+        propertyDTOs[2].setName("userName");
+        propertyDTOs[2].setValue(DB_USER_NAME);
+
+        propertyDTOs[3].setName("password");
+        propertyDTOs[3].setValue(DB_USER_PASSWORD);
+
+        propertyDTOs[4].setName("PasswordJavaRegEx");
+        propertyDTOs[4].setValue("^[\\S]{5,30}$");
+
+        propertyDTOs[5].setName("UsernameJavaRegEx");
+        propertyDTOs[5].setValue("^[\\S]{5,30}$");
+
+        propertyDTOs[6].setName("Disabled");
+        propertyDTOs[6].setValue("false");
+
+        propertyDTOs[7].setName("PasswordDigest");
+        propertyDTOs[7].setValue("SHA-256");
+
+        propertyDTOs[8].setName("StoreSaltedPassword");
+        propertyDTOs[8].setValue("true");
+
+        propertyDTOs[9].setName("SCIMEnabled");
+        propertyDTOs[9].setValue("true");
+
+        UserStoreDTO userStoreDTO = userStoreConfigAdminServiceClient
+                .createUserStoreDTO(jdbcClass, DOMAIN_ID, propertyDTOs);
+        userStoreConfigAdminServiceClient.addUserStore(userStoreDTO);
+        Thread.sleep(5000);
+        userStoreConfigUtils.waitForUserStoreDeployment(userStoreConfigAdminServiceClient, DOMAIN_ID);
+    }
+
+    @Test(dependsOnMethods = "addSecondaryUserStore")
+    public void testSCIM1CreateGroup() throws Exception {
+
+        JSONObject userJSON = scim1Group.getRoleJSON(inputFileName);
+        HttpResponse response = scim1Group.provisionGroup(client, userJSON, username, password);
+
+        assertEquals(response.getStatusLine().getStatusCode(), HttpStatus.SC_CREATED,
+                "Group has not been created successfully");
+
+        JSONObject returnedUserJSON = getJSONFromResponse(response);
```

Comment:
Since it group change variable as returnedGroupJSON, check other places as well

Reply:
changed